oeps/oep-0002-bp-repo-metadata.rst

```diff
@@ -2,24 +2,25 @@
 OEP-2: Repository Metadata
 ==========================
 
-+---------------+-------------------------------------------+
-| OEP           | :doc:`OEP-2 <oep-0002-bp-repo-metadata>`  |
-+---------------+-------------------------------------------+
-| Title         | Repository Metadata                       |
-+---------------+-------------------------------------------+
-| Last-Modified | 2019-10-29                                |
-+---------------+-------------------------------------------+
-| Author        | Calen Pennington <cale@edx.org>           |
-|               | Feanil Patel <feanil@edx.org>             |
-+---------------+-------------------------------------------+
-| Arbiter       | Calen Pennington <cale@edx.org>           |
-+---------------+-------------------------------------------+
-| Status        | Accepted                                  |
-+---------------+-------------------------------------------+
-| Type          | Best Practice                             |
-+---------------+-------------------------------------------+
-| Created       | 2016-04-21                                |
-+---------------+-------------------------------------------+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-2 <oep-0002-bp-repo-metadata>`
+   * - Title
+     - Repository Metadata
+   * - Last Modified
+     - 2019-11-14
```

Comment:
Minor point - should this be updated to 11/22 now?

Reply:
I can go either way on this.  Do reformatting changes require updating the timestamp?  I'm also not sending out reformatting changes for a wider review.  Git already catches these iterative timestamps, right?